executor/app/src/compute_horde_executor/executor/job_runner.py

```diff
@@ -245,113 +237,26 @@ async def start_job(self) -> AsyncGenerator[None, Any]:
-
-                assert self.executor_certificate is not None
-                # check that the job is ready to serve requests
-                ip = await get_docker_container_ip(self.nginx_container_name, bridge_network=True)
-                logger.debug(f"Checking if streaming job is ready at http://{ip}/health")
-                job_ready = await check_endpoint(
-                    f"http://{ip}/health",
-                    WAIT_FOR_STREAMING_JOB_TIMEOUT,  # TODO: TIMEOUTS - Remove timeout?
-                )
-                if not job_ready:
-                    raise JobError("Streaming job health check failed")
-
-            yield
-
-            try:
-                # Support for the legacy single-timeout
-                docker_process_timeout = (
-                    self.initial_job_request.executor_timing.execution_time_limit
-                    if self.initial_job_request.executor_timing
-                    else self.initial_job_request.timeout_seconds
-                )
-                logger.debug(
-                    f"Waiting {docker_process_timeout} seconds for job container to finish"
-                )
-                result = await asyncio.wait_for(
-                    job_process.communicate(),
-                    timeout=docker_process_timeout,
-                )
-                logger.debug(f"Job container exited with return code {job_process.returncode}")
-                self.execution_result = ExecutionResult(
-                    timed_out=False,
-                    return_code=job_process.returncode,
-                    stdout=result[0].decode(),
-                    stderr=result[1].decode(),
-                )
-
-            except TimeoutError:
-                logger.debug("Job container timed out")
-                stdout = (await job_process.stdout.read()).decode() if job_process.stdout else ""
-                stderr = (await job_process.stderr.read()).decode() if job_process.stderr else ""
-                self.execution_result = ExecutionResult(
-                    timed_out=True,
-                    return_code=None,
-                    stdout=stdout,
-                    stderr=stderr,
-                )
-
-            if self.is_streaming_job:
-                # stop the associated nginx server
-                try:
-                    await asyncio.sleep(1)
-                    process = await asyncio.create_subprocess_exec(
-                        "docker",
-                        "stop",
-                        self.nginx_container_name,
-                    )
-                    try:
-                        await asyncio.wait_for(process.wait(), DOCKER_STOP_TIMEOUT_SECONDS)
-                    except TimeoutError:
-                        process.kill()
-                        raise
-                except Exception as e:
-                    logger.error(f"Failed to stop Nginx: {e}")
+            yield await self.after_start_job()
```

Comment:
shouldn't this be 
```suggestion
            await self.after_start_job()
            yield 
```
?
I'm not sure the value returned by `self.after_start_job` should be passed up as the value of this async context manager. previously it was none.